### PR TITLE
Add `-dontobfuscate`  to Macrobenchmark Sample

### PR DESCRIPTION
**Pull request by tikurahul**

### CODE REVIEW

File: MacrobenchmarkSample/app/proguard-rules.pro [hunk 1]
```diff
@@ -18,4 +18,11 @@
 
 # If you keep the line number information, uncomment this to
 # hide the original source file name.
-#-renamesourcefileattribute SourceFile
\ No newline at end of file
+#-renamesourcefileattribute SourceFile
+
+# Turn off obfuscations for Baseline Profile Generation
+# This is because Baseline Profile rules participate in the build and will eventually get
```
Comment (ChrisCraik) [inline MacrobenchmarkSample/app/proguard-rules.pro:24]: Suggest rewording:

Baseline profiles must be generated from an unobfuscated build, so that the text format matches source. Subsequently, when compiled into binary format within your release APK, they will account R8 obfuscation mapping.

File: MacrobenchmarkSample/app/proguard-rules.pro [hunk 2]
```diff
@@ -18,4 +18,11 @@
...
+# obfuscated with the same obfuscation map that is being used for the APK. We don't want the
+# rules that are generated by BaselineProfileRule to be obfuscated on their own. We want the APK and
+# the profile rules to be obfuscated together.
+-dontobfuscate
```
Comment (ChrisCraik) [inline MacrobenchmarkSample/app/proguard-rules.pro:28]: can we have this as a separate `do-not-obfuscate.pro` that's only used in the benchmark build type? that will match what external devs would do more closely (hopefully we don't need a separate variant for that)

Comment (mlykotom) [reply MacrobenchmarkSample/app/proguard-rules.pro:28]: Agree with separate proguard file .. maybe `proguard-rules-benchmark.pro` not to confuse with the name?

**Comment by keyboardsurfer** [comment]
Superseded by #134.
